Examples/DemoPlacementViewController.swift

```diff
@@ -29,9 +31,10 @@ class DemoPlacementViewController: UIViewController {
         sceneView.defaultCameraController.inertiaEnabled = true
         sceneView.showsStatistics = true
 
-        //Set up initial terrain and materials
-        let terrainNode = TerrainNode(minLat: 50.044660402821592, maxLat: 50.120873988090956,
-                                      minLon: -122.99017089272466, maxLon: -122.86824490727534)
+        let diffLat = 50.120873988090956 - 50.044660402821592
```

Comment:
Maybe add a comment about where this location is?

Reply:
This lat/lon doesn't appear to be different from the original, adding a comment anyway